gpMgmt/sbin/gpgetstatususingtransition.py

```diff
@@ -35,8 +35,13 @@
 POSTMASTER_MIRROR_VERSION_DETAIL_MSG = "- VERSION:"
 
 def _get_segment_status(segment):
-    cmd = base.Command('pg_isready for segment',
+    PGDATABASE = os.getenv('pgdatabase')
+    if (PGDATABASE == 'None'):
```

Comment:
It seems that this condition is always false. I have no idea, how [this test](https://github.com/arenadata/gpdb/blob/74fed1a90048624626eba52d210e9edc93f993b4/gpMgmt/test/behave/mgmt_utils/gpstate.feature#L560-L566) may be passed.